Cartfile

```diff
@@ -1,4 +1,4 @@
-binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" ~> 20.1.0
-binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" ~> 80.0
+binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.json" == 21.0.0-rc.1
```

Comment:
Pinning this version is unfortunate but necessary for consistency [with SPM](https://github.com/mapbox/mapbox-navigation-native-ios/blob/beb14b30bdb3bcde32544756f1867d82b6d758bf/Package.swift#L8) and CocoaPods. But we’ll need to make sure to go back to a tadpole operator once MapboxCommon v21.0.0 is released and MapboxNavigationNative depends on it.